.eslintrc.js

```diff
@@ -27,5 +20,5 @@ module.exports = {
     ],
     'no-param-reassign': 'off',
   },
-  ignorePatterns: ['!.eslintrc.js', './test/*.js', 'dist'],
+  ignorePatterns: ['!.eslintrc.js', 'test/*.js', 'dist'],
```

Comment:
should we add back the `coverage` and `*.d.ts` from the removed `.eslintignore` . And the `node_modules` ignore should maybe go in `tsconfig`?

Reply:
Ah, good catch! But it looks like all of these are fine to ignore. `node_modules` is [ignored by default](https://eslint.org/docs/user-guide/configuring/ignoring-code#the-eslintignore-file), and we don't have any `coverage` directory or `*.d.ts` files in this repo.